tests/test_data/projects/snowpark_function_external_access/snowflake.yml

```diff
@@ -0,0 +1,16 @@
+definition_version: 1
+functions:
+  - name: func1
+    handler: "app.func1_handler"
+    signature:
+      - name: "a"
+        type: "string"
+      - name: "b"
+        type: "variant"
+    returns: string
+    external_access_integrations:
+      - "external_1"
+      - "external_2"
+    secrets:
+      cred: "cred_name"
+      other: "other_name"
```

Comment:
Should we consider adding external access to template?

Reply:
Added